values.yaml

```
@@ -198,6 +198,25 @@ global:
       secretName: null
       secretKey: null
 
+  # Settings related to federating with other Consul datacenter.
+  federation:
+    # **Experimental: Currently this is only available in Consul 1.8.0-beta.**
+    # If enabled, this datacenter will be federation-capable. Only federation
+    # through mesh gateways is supported.
+    # Mesh gateways and servers will be configured to allow federation.
+    # Requires tls.enabled, meshGateway.enabled and connectInject.enabled
```

When you read it together with `meshGateway.enabled` and `connectInject.enabled`, it could be interpreted as a property that's outside of `global`
```suggestion
    # Requires global.tls.enabled, meshGateway.enabled and connectInject.enabled
```